config/locales/devise/es.yml

```diff
@@ -136,7 +136,7 @@ es:
         text_html: >
           Si no puede usar su aplicación de autenticador ahora, puede %{sms_link}
           O %{voice_link}.
-        voice_link_text: con una llamada telefónica
+        voice_link_text_html: con una llamada telefónica
```

Comment:
this link is not as accessible as the english version because there is no hidden span.